### PR TITLE
Update a few last minute descriptions

### PR DESCRIPTION
**Pull request by erikkerber**

### CODE REVIEW

File: RWDevCon/Supporting Files/conferences.json [hunk 1]
```diff
@@ -650,7 +650,7 @@
...
+        "description": "Web Kit\u2019s WKWebView is Apple's replacement for the limited UIWebView control and it\u2019s more than just for "web browser apps". Introduced with iOS 8, WKWebView has the same power as the built-in Safari App but as a public API. Performance optimizations, such as the Nitro JIT JavaScript engine, make WKWebView a powerful platform to display HTML and CSS (and to interpret javascript) easily within your apps. With Web Kit you can re-use content coming from a web CMS or local html file, and more! In this session, you will implement WKWebView to load both local and remote content that interacts with your app seamlessly. You will learn how to properly use CSS and custom fonts with your local content, how to intercept hyperlink taps (to redirect to in-app actions), and more.",
```
Comment (sberrevoets) [inline RWDevCon/Supporting Files/conferences.json]: The quotations here will break the JSON

Comment (erikkerber) [reply RWDevCon/Supporting Files/conferences.json]: Knew I'd miss 1 or 7

**Comment by erikkerber** [comment]
@sberrevoets Fixed all of them (I think)

**Comment by sberrevoets** [comment]
Only 1 way to find out!

**Pull request merged**